tests/DocumentationParserTest.php

```diff
@@ -325,21 +320,21 @@ public function testApiLinks()
         $page_version = $this->page->getVersion();
 
         // expected url format resulting from rewriting api shortcode links
-        $url_format = '<a href="http://api.silverstripe.org/search/lookup/?q=%s&version='.$page_version.'&module=documentationparsertest">%s</a>';
+        $html_format = '&nbsp;<a href="http://api.silverstripe.org/search/lookup/?q=%s&version='.$page_version.'&module=documentationparsertest">%s</a>';
```

Comment:
There is something strange going on that I haven't quite pinned down: the first and last characters sometimes get gobbled up. The &nbsp; puts a space back in front. And, in the instructions, I say that one must put a space at the end of the shortcode. This is not satisfactory but works but it works most of the time. I am trying to get backticks to work with the api links like this `[title](api:link)` as they do with ordinary markdown links of the form `[title)(link)`. I'll try to get this sorted...